package.json

```diff
@@ -4,9 +4,12 @@
   "private": true,
   "scripts": {
     "build": "tsc",
-    "start": "nodemon -e ts --exec \"(npm run lint -- --force && tsc && node build/server.js) || true\"",
+    "start": "npm run lint && npm run build && npm-run-all --parallel build:watch lint:watch serve:watch",
```

Comment:
Unfortunately [tsc-watch](https://github.com/gilamran/tsc-watch) triggers `onSuccess` on every successful compilation, not only the first one. So naive use of it solves the double triggering on start-up, but crushes the pipe on change. But anyway it is a good tool, and probably will be needed later. And I'm still trying to find away to solve this issue.